src/components/Messages/MessageFolder.jsx

```diff
@@ -28,6 +28,7 @@ import { EmptyListNotification, LoadingAnimation } from '../Notification';
  * @param {messageFolderProps} Props - Component props for MessageFolder
  * @returns {React.JSX.Element} React component for MessageFolder
  */
+
```

Comment:
The removal of the spacing was intentional. It helps JSDoc associate the comment with the component directly below it.

https://github.com/codeforpdx/PASS/assets/14917816/dc7dda30-cef2-476d-a7dd-31448728bf01